### PR TITLE
deal with namespaces, XMLs with no start/end time

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

File: R/01-webdata-obj.R [hunk 1]
```diff
@@ -108,7 +115,11 @@ setMethod("webdata", signature("character"), function(.Object=c("prism",  "iclus
 setMethod("webdata", signature("geojob"), function(.Object, ...) {
   xmlVals <- inputs(xmlParse(xml(.Object)))
   url <- xmlVals[["DATASET_URI"]]
-  times <- c(xmlVals[["TIME_START"]], xmlVals[["TIME_END"]])
+  if(any(grepl(pattern = "TIME_",  x = names(xmlVals)))){ #underscore is important
```
Comment (jiwalker-usgs) [inline R/01-webdata-obj.R]: I think this also needs to deal with the case where there is a START but no END, or an END with now START.

File: R/inputs-webprocess.R [hunk 2]
```diff
@@ -57,7 +57,7 @@ setMethod(f = "inputs",signature = "XMLAbstractDocument",
           definition = function(.Object, ...){
             inputXpath <- "//wps:Execute/wps:DataInputs/wps:Input"
             
-            inputs <- getNodeSet(.Object, inputXpath)
+            inputs <- getNodeSet(.Object, inputXpath, namespaces = pkg.env$NAMESPACES)
```
Comment (jiwalker-usgs) [inline R/inputs-webprocess.R]: We probably need to spread to this to other Xpath calls.

File: R/01-webdata-obj.R (hunk 1 above)
Comment (jread-usgs) [inline R/01-webdata-obj.R]: agreed - we need to handle c(`start`, NA), c(NA, `end`) , and c(NA, NA)

**Comment by jread-usgs** [comment]
@wdwatkins not sure what is going on w/ the travis test installing `ggmap`.

**Comment by wdwatkins** [comment]
Looks like just an internet issue during the download?

File: R/inputs-webprocess.R (hunk 2 above)
Comment (wdwatkins) [inline R/inputs-webprocess.R]: Should this be used with all `getNodeSet` calls?

Comment (jiwalker-usgs) [inline R/inputs-webprocess.R]: Most if not all

Comment (wdwatkins) [inline R/inputs-webprocess.R]: I'm getting errors in most places where I add this (calls that use the default `namespaces`).

**Comment by jread-usgs** [comment]
@wdwatkins 
```
1. Error: show datagroup (@test-show_object.R#23) ------------------------------
cannot find defined namespace(s) with prefix(es) srv, gmd, gco
1: expect_output(show(query("webdata"))) at testthat/test-show_object.R:23
2: capture_output(object)
3: capture_output_as_vector(code, print)
4: with_sink(temp, withVisible(code))
5: withVisible(code)
6: show(query("webdata"))
7: query("webdata")
```

**Comment by wdwatkins** [comment]
☝️ not sure why that was happening, it was fine locally.  Maybe there was a connection problem with the urls where the namespaces are defined?

**Comment by wdwatkins** [comment]
@jread-usgs or @jiwalker-usgs merge?

**Pull request merged**